lib/tasks/dm.rake

```diff
@@ -25,4 +25,28 @@ namespace :dm do
     Rake::Task['dm:full_import'].execute
   end
 
+  # rails dm:dummy_data
+  desc 'Import dummy data'
+  task :dummy_data => :environment do
+    puts '==> Importing dummy data'
+
+    dummy_data = "#{Rails.root}/tmp/practices_dummy_data.csv"
+
+    CSV.foreach(dummy_data, headers: true) do |row|
+      puts "==> Importing practice '#{row['project_name']}'"
+      practice = Practice.create(
+        name: row['project_name'],
+        tagline: row['project_name'],
+        date_initiated: row['start_date'],
+        description: row['purpose'],
+        summary: row['approach'],
+        initiating_facility: row['business_office_sponsor']
+      )
+
+      practice.approved = true
+      practice.published = true
```

Comment:
how come these aren't in the `create` arguments?

Reply:
Ah, no reason, a victim of too much refactoring. Will push an update.